README.md

```diff
@@ -98,6 +98,42 @@ function visible() {
 
 When your element is near `300px` of the viewport, you get your callback / true result.
 
+### Specifying debounce value
+
+By default, debounce value for scroll and resize events is 15ms.
+
+You can change that with `debounce` param.
+
+```js
+var inViewport = require('in-viewport');
+var elem = document.getElementById('myFancyDiv');
+
+inViewport(elem, { debounce: 300 }, visible);
+
+function visible() {
+  alert('myfancyDiv is visible in the `customContainer`!');
+}
+```
+
+### Failsafe check
+
+By default, inViewport does a failsafe to handle display manipulation that does not throw an event. It works with a `setInterval` performed every 150ms.
+
+Usecase: a hidden parent containing elements; when the parent becomes visible, we have no event that the children became visible.
+
+If you don’t want to run that check constantly, turn it off with `failsafe` param.
```

Comment:
`setInterval` is called every 150ms which is in most cases unecessary. It is failsafe after all and it covers edge case from what I can see. Maybe it should be disabled by default and left to developer to handle it by themselves? Same thing is with MutationObservers—it is handled in browsers which support them, but for all others, developers are instructed to polyfill support.